client/my-sites/checkout/composite-checkout/wpcom/components/test/lib/fixtures.ts

```diff
@@ -0,0 +1,362 @@
+/**
+ * External Dependencies
+ */
+import moment from 'moment';
+
+export const responseCartWithRenewal = {
+	products: [
+		{
+			product_id: 74,
+			product_name: '.live Domain Registration',
+			product_slug: 'dotlive_domain',
+			product_cost_display: '$380',
+			product_cost_integer: 38000,
+			meta: 'user-personal-site-test-1234.live',
```

Comment:
I think this is the wrong domain.  (Super-minor, but I noticed it only because I was trying to understand the tests and got confused about why it was adding something to the cart that I thought was already there).